t/run/switches.t

```diff
@@ -115,7 +115,7 @@ SKIP: {
 
     local $ENV{'LC_ALL'} = 'C'; # Keep the test simple: expect English
     local $ENV{LANGUAGE} = 'C';
-    setlocale(LC_ALL, "C");
+    setlocale('LC_ALL', "C");
```

Comment:
Alternatively the fat comma could have helped there.
This is a good fix

Reply:
True, but `LC_ALL` is single-quoted in this section of the code.